### PR TITLE
Improve resource manager test coverage

### PR DESCRIPTION
**Pull request by luckyarthur**
## Why are the changes needed?

the resource manager test coverage is 64.04%, which is behind 80%, this PR improves the coverage

## What changes were proposed in this pull request?

add more test case coverage to resource manager in this pull request

## How was this patch tested?

tests were run after every commit

## Check all the applicable boxes 

- [ ] I updated the documentation accordingly.
- [x] All new and existing tests passed.
- [x] All commits are signed-off.

### CODE REVIEW

File: flyteadmin/pkg/manager/impl/resources/resource_manager_test.go [hunk 1]
```diff
@@ -54,6 +54,13 @@ func TestUpdateWorkflowAttributes(t *testing.T) {
 	_, err := manager.UpdateWorkflowAttributes(context.Background(), request)
 	assert.Nil(t, err)
 	assert.True(t, createOrUpdateCalled)
+
+	request = &admin.WorkflowAttributesUpdateRequest{
+		Attributes: &admin.WorkflowAttributes{},
+	}
+	_, failError := manager.UpdateWorkflowAttributes(context.Background(), request)
+	assert.Error(t, failError)
```
Comment (troychiu) [inline flyteadmin/pkg/manager/impl/resources/resource_manager_test.go:62]: Can we also make sure it returns the right error?

File: flyteadmin/pkg/manager/impl/resources/resource_manager_test.go [hunk 2]
```diff
@@ -181,6 +188,33 @@ func TestGetWorkflowAttributes(t *testing.T) {
...
 		},
 	}, response))
+
+	db.ProjectRepo().(*mocks.MockProjectRepo).GetFunction = func(
+		ctx context.Context, projectID string) (models.Project, error) {
+		return models.Project{}, errors.NewFlyteAdminError(codes.NotFound, "validationError")
+	}
+
+	_, validationError := manager.GetWorkflowAttributes(context.Background(), request)
+	assert.Error(t, validationError)
```
Comment (troychiu) [inline flyteadmin/pkg/manager/impl/resources/resource_manager_test.go:198]: same here

File: flyteadmin/pkg/manager/impl/resources/resource_manager_test.go (hunk 1 above)
Comment (troychiu) [reply flyteadmin/pkg/manager/impl/resources/resource_manager_test.go:62]: and other similar places

Comment (luckyarthur) [reply flyteadmin/pkg/manager/impl/resources/resource_manager_test.go:62]: all new added error test cases have been error type and error message checked in the code

**Review by eapolinario — APPROVED** [review]

**Pull request merged**